datafusion/core/src/physical_plan/file_format/parquet.rs

```diff
@@ -900,6 +900,8 @@ pub async fn plan_to_parquet(
 
 #[cfg(test)]
 mod tests {
+    // See also `parquet_exec` integration test
+
     use super::*;
```

Comment:
@Ted-Jiang  / @tustvold  / @thinkharderdev  what do you think about moving the remaining tests in this module into the integration test?

Reply:
I think some of them may make use of crate private functions, otherwise I have no strong feeling either way

Reply:
👍  yes I think the pruning ones in particular do so -- I would likely need to refactor them

Reply:
No objection from me. I think we can probably leave the pruning tests as unit tests here and just move the more "integration-y" tests?